gift_compassion/data/process_gift_cron.xml

```diff
@@ -1,12 +1,13 @@
 <odoo>
-    <data noupdate="1">
+    <data>
+<!--    <data noupdate="1">-->
         <record id="process_gift_cron" model="ir.cron">
             <field name="name">Process gifts</field>
             <field name="interval_number">1</field>
             <field name="interval_type">days</field>
             <field name="numbercall">-1</field>
             <field name="function">process_gifts_cron</field>
```

Comment:
This should be changed to `code` and add a field `<field name="state">code</field>`
```suggestion
            <field name="code">model.process_gifts_cron()</field>
```